cmd/kgo-repeater/main.go

```diff
@@ -32,7 +32,7 @@ var (
 	password           = flag.String("password", "", "SASL password")
 	enableTls          = flag.Bool("enable-tls", false, "Enable use of TLS")
 	brokers            = flag.String("brokers", "localhost:9092", "comma delimited list of brokers")
-	topic              = flag.String("topic", "", "topic to produce to or consume from")
+	topics             = flag.String("topics", "", "topic(s) to produce to or consume from")
```

Comment:
This is going to have a pretty large blast radius.  Ducktape test services that utilize this, the LRC load clients, maybe others.  Is there another way of providing this?  Maybe passing `-topic` multiple times?

Reply:
agree to about blast radious. multiple -topic is good, or have both options, or maybe just pass the comma separated list to `-topic`? if "a,b" and the backwards compat single version "a" would both parse correctly.

Reply:
Went with having `-topic` and `-topics` work the same way. I guess in the future when all of the programs invoking this are migrated to using `-topics`, we can deprecate `-topic`